viz.yaml

```diff
@@ -256,24 +256,26 @@ publish:
     id: map-section
     publisher: section
     template: simplefigure
-    depends:
+    depends: siteText
```

Comment:
this should be on the next line with something like `text: siteText`  also, siteText will need to be defined in yaml